engine/lite/k8sDeploymentManager/src/test/scala/pl/touk/nussknacker/k8s/manager/ingress/IngressPreparerTest.scala

```diff
@@ -0,0 +1,102 @@
+package pl.touk.nussknacker.k8s.manager.ingress
+
+import com.typesafe.config.ConfigFactory
+import com.typesafe.config.ConfigValueFactory.{fromIterable, fromMap}
+import org.scalatest.funsuite.AnyFunSuite
+import org.scalatest.matchers.should.Matchers.convertToAnyShouldWrapper
+import pl.touk.nussknacker.engine.api.process.{ProcessId, ProcessName, VersionId}
+import pl.touk.nussknacker.engine.api.{LiteStreamMetaData, ProcessVersion, RequestResponseMetaData, TypeSpecificData}
+import skuber.ObjectMeta
+import skuber.networking.v1.Ingress
+
+import scala.collection.JavaConverters._
+
+class IngressPreparerTest extends AnyFunSuite {
```

Comment:
I wonder if we can have normal, integration test. It just need `disableLoadbalancer` to be false in k3d config - the same as we have in quickstart repo. After that calling http://localhost:8081/slug should be just fine. I'm only worry if there won't be some troubles with port collisions or sth like this. WDYT? On the other hand test in quickstart repo works fine.

Reply:
I'm still experimenting on this, will let you know soon